slides/advanced-02-tuning-hyperparameters.qmd

```diff
@@ -520,6 +532,7 @@ lgbm_wflow <- workflow(hotel_rec, lgbm_spec)
 ```{r tuning} 
 #| label: lgb-grid-tune
 #| cache: true
+#| code-line-numbers: "2|"
 set.seed(9)
 ctrl <- control_grid(save_pred = TRUE, parallel_over = "everything")
```

Comment:
Do we want to show `parallel_over = "everything"` here? It might be interpreted as "switching on parallelism", especially since we only talk about how to really do that a little later in the slides